components/NavDrawer/NavDrawer.tsx

```diff
@@ -128,8 +128,8 @@ const DrawerTitle = React.forwardRef<
       {...props}
     />
 
-    <DrawerClose>
-      <X />
+    <DrawerClose data-testid="drawer-close">
+      <X data-testid="drawer-close-icon" />
```

Comment:
BEM naming convention please.
```suggestion
      <X data-testid="drawer-close__icon" />
```